apps/climatemappedafrica/src/payload/globals/HURUMap/index.js

```diff
@@ -15,6 +19,36 @@ const HURUMap = {
       relationTo: ["pages"],
       maxDepth: 1,
       required: true,
+      admin: {
+        description:
+          "The page to use as the Explore page. It will contain the interactive map.",
+      },
+    },
+    {
```

Comment:
1. If this is here (which I think is correct), shouldn't we remove these from Hero block? Otherwise, editors have to set this twice.
2. Should't you include an option to indicate whether pinning should be enabled or not at initial location? I think in PesaYetu, pinning/comparison is enabled for KE.

Reply:
> If this is here (which I think is correct), shouldn't we remove these from Hero block? Otherwise, editors have to set this twice.

I also agree with this @kilemensi. Initially, we thought the map in the hero could be different, but thinking about it further, it should be the same